examples/miral-shell/shell_main.cpp

```diff
@@ -55,6 +56,8 @@ int main(int argc, char const* argv[])
 
     ExternalClientLauncher external_client_launcher;
 
+    std::string terminal_cmd{"weston-terminal"};
```

Comment:
Personal preference:

Top of file:
```
std::string const default_terminal_cmd{"weston-terminal"};
```

here: 
```
std::experimental::optional<std::string> requested_terminal_cmd;
```

When used:
```
auto terminal_command = requested_terminal_cmd.value_or(default_terminal_command);
```

Feel free to disregard, but I think it makes it a little more readable.